sdk/eventhub/azure-eventhub/azure/eventhub/aio/_producer_client_async.py

```diff
@@ -11,8 +11,11 @@
 from ..exceptions import ConnectError, EventHubError
 from ._client_base_async import ClientBaseAsync
 from ._producer_async import EventHubProducer
+from .._producer_client import validate_outgoing_event_data
 from .._constants import ALL_PARTITIONS
 from .._common import EventDataBatch, EventData
+from .._configuration import PartitionPublishingConfiguration
+from .._utils import validate_producer_client_partition_config
 
```

Comment:
same comments from sync apply to async

Reply:
sorry.. I'm a bit losing the context (too many comments) -- which comments are you referring to?

Reply:
sorry, should have stated explicitly. It was just the same naming comments, like `partition_config` instead of `partition_configs` etc. Looks like those have all been resolved so you're good!